source/AI.cpp

```diff
@@ -1620,7 +1632,9 @@ void AI::MoveEscort(Ship &ship, Command &command) const
 	{
 		ship.SetTargetSystem(nullptr);
 		ship.SetTargetStellar(parent.GetTargetStellar());
-		MoveToPlanet(ship, command);
+		// Escorts always move at the fastest speed possible in service of their parent.
+		double cruiseSpeed = 0.;
+		MoveToPlanet(ship, command, cruiseSpeed);
```

Comment:
Yes, this PR currently only changes the behaviour of the parent. The escorts both in master as well as currently in this PR just fly to the planet as fast as they can when the parent is landing.

I agree that it would seem better to have the escorts stick to the parent. I'll have a look if changing the escorts behaviour could be combined into this PR or if it is better to create a separate PR for that (the code overlap looks minimal).

Reply:
Handled in #5621